lfsapi/lfsapi.go

```diff
@@ -90,6 +90,14 @@ func NewClient(osEnv env, gitEnv env) (*Client, error) {
 	return c, nil
 }
 
+func (c *Client) GitEnv() env {
```

Comment:
Not really, since you pass the os and git envs to `NewClient()`. If transfer adapters get extracted to a separate package, I think it will need to be public.  I'll unexport for now, and we can export later if necessary.